.github/workflows/playwright.yml

```
@@ -160,6 +174,11 @@ jobs:
       env:
         CI: true
         token: ${{ secrets.KITTYCAD_API_TOKEN_DEV }}
+    - name: send to axiom
+      if: always()
+      shell: bash
+      run: |
+        node playwrightProcess.mjs | tee /tmp/playwright-fails.log
```

So it looks like the vector.toml is specifically looking at the `/tmp/github-actions.log` file. So, instead of `tee /tmp/playwright-fails.log` this should be:
```suggestion
        node playwrightProcess.mjs | tee /tmp/github-actions.log
```